src/components/attendee/VideoComms/types.tsx

```diff
@@ -0,0 +1,94 @@
+import Twilio from "twilio-video";
+
+export enum VideoCommsStatus {
+  Disconnected = "DISCONNECTED",
+  Connecting = "CONNECTING",
+  Connected = "CONNECTED",
+  Errored = "ERRORED",
+}
+
+export interface VideoCommsProviderProps {
+  userId: string;
+  children: React.ReactNode;
+}
+
+interface Track {
+  kind: "video" | "audio";
+  id: string;
+  enabled: boolean;
+}
+
+export enum VideoSource {
+  Webcam = "WEBCAM",
+  Screenshare = "SCREENSHARE",
+}
+
+export interface VideoTrack extends Track {
+  attach: (el: HTMLVideoElement) => void;
+  detach: () => void;
+  kind: "video";
+  twilioTrack: Twilio.VideoTrack;
+  sourceType: VideoSource;
+}
+
+export interface AudioTrack extends Track {
+  attach: (el: HTMLAudioElement) => void;
+  detach: () => void;
+  kind: "audio";
+  twilioTrack: Twilio.AudioTrack;
+}
+
+export interface Participant {
+  /**
+   * ID of the participant - used for communication with the video platform
+   */
+  id: string;
+
+  /**
+   * Video tracks
+   */
+  videoTracks: VideoTrack[];
+
+  /**
+   * Audio tracks
+   */
+  audioTracks: AudioTrack[];
+}
+
+type JoinChannelFunc = (
+  userId: string,
+  channelId: string,
+  enableVideo: boolean,
+  enableAudio: boolean
+) => void;
+type DisconnectFunc = () => void;
+type ShareScreenFunc = () => void;
+
+export interface LocalParticipant extends Participant {}
```

Comment:
No need of both I guess.

```suggestion
export interface LocalParticipant extends Participant {}
```

Maybe something similar like the previous suggestion:
`export type Participant = LocalParticipant | RemoteParticipant;`